src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -62,6 +62,9 @@
       <AssemblyInfoLines Condition="'$(AssemblyComVisible)'!=''" Include="[assembly:System.Runtime.InteropServices.ComVisible($(AssemblyComVisible))]" />
       <AssemblyInfoLines Condition="'$(SkipFrameworkAssemblyMetadata)' != 'true'"
         Include="[assembly:System.Reflection.AssemblyMetadata(&quot;%(AssemblyMetadata.Identity)&quot;, &quot;%(AssemblyMetadata.Value)&quot;)]" />
+
+      <AssemblyInfoLines Condition="'$(IncludeDllSafeSearchPathAttribute)'=='true'" 
```

Comment:
The later. Just set IncludeDllSafeSearchPathAttribute=true if IsDotNetFrameworkProductAssembly=true.